uberenv.py

```diff
@@ -464,10 +594,9 @@ def install(self):
         install_cmd = "spack/bin/spack "
         if self.opts["ignore_ssl_errors"]:
             install_cmd += "-k "
-        if not self.opts["install"]:
-            install_cmd += "dev-build -d {} -u {} ".format(self.pkg_src_dir,self.pkg_final_phase)
-        else:
-            install_cmd += "install "
+        install_cmd += "dev-build -d {0} ".format(self.pkg_src_dir)
+        if not self.opts["install"] and self.pkg_final_phase:
+            install_cmd += "-u {0} ".format(self.pkg_final_phase)
```

Comment:
Again, I can’t remember why I wanted to keep `install` here, instead of dev-build.
I know that I think about it, it makes sense that one would want to install the development version. But I know there is a snag.